_posts/when-too-many-choices-are-harmful.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+author: Mauro Servienti
+title: "When too many choices are harmful"
+header_image: /img/posts/when-too-many-choices-are-harmful/header.jpg
+synopsis: "I tend to be lazy and pragmatic when it comes to choices. It's also trivial that being exposed to too many options stresses me producing a negative result. I end up refusing to choose."
+tags:
+- personal
+- processes
+---
+
+I prefer restaurants with short menus rather than encyclopedic ones. When sitting at a table, faced with pages of choices and the pressure the waiter puts on me, I urge to wake up and escape.
```

Comment:
```suggestion
I prefer restaurants with short menus rather than encyclopedic ones. When sitting at a table, faced with pages of choices and the pressure the waiter puts on me, I have the urge to wake up and escape.
```